bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1635,111 +1635,106 @@ That is, the atmosphere (rather than the ocean) melts the ice.
 </entry>
 
 <!-- ========================================================================================  -->
-<!-- ndepdyn streams Namelist (only used when bgc=cn or bgc                                    -->
+<!-- ndepdyn streams Namelist (only used when bgc=cn or bgc)                                   -->
 <!-- ========================================================================================  -->
 
-<entry id="stream_year_first_ndep" type="integer" category="datasets"
-       group="ndepdyn_nml" valid_values="" >
+<entry id="stream_ndep_year_first" type="integer" category="datasets" group="ndepdyn_nml" valid_values="" >
 First year to loop over for Nitrogen Deposition data
 </entry>
 
-<entry id="stream_year_last_ndep" type="integer" category="datasets"
-       group="ndepdyn_nml" valid_values="" >
+<entry id="stream_ndep_year_last" type="integer" category="datasets" group="ndepdyn_nml" valid_values="" >
 Last year to loop over for Nitrogen Deposition data
 </entry>
 
-<entry id="model_year_align_ndep" type="integer" category="datasets"
-       group="ndepdyn_nml" valid_values="" >
+<entry id="stream_ndep_year_align" type="integer" category="datasets" group="ndepdyn_nml" valid_values="" >
 Simulation year that aligns with stream_year_first_ndep value
 </entry>
 
-<entry id="stream_fldfilename_ndep" type="char*256" category="datasets"
-       input_pathname="abs" group="ndepdyn_nml" valid_values="" >
```

Comment:
I don't think the input_pathname part shoudl be removed here. That is basically the way to indicate it's a file.